crates/polars-io/src/csv/write.rs

```diff
@@ -15,6 +15,8 @@ pub enum QuoteStyle {
     Necessary,
     /// This puts quotes around all fields that are non-numeric. Namely, when writing a field that does not parse as a valid float or integer, then quotes will be used even if they aren’t strictly necessary.
     NonNumeric,
+    /// Never quote any fields, even if it would usually be considered necessary.
```

Comment:
Shall we explicitly name it can produces invalid CSV files: `"even if it would produce invalid CSV data"`

Reply:
May as well leave no doubt!  ;)